alexherlanda/db8-app
# Post Event UI

## Abstract 
Adds a post flow to create an event card in an interactive mode. Also adds an enhance on the cards to visualize a link collection rather than a single link. 

## Purpose
- Allowing the user to post an event in the platform with no intermediaries 
- Allowing the user to see a collection of links (masters, open, adjudicators, etc) 

## Summary 
- Adds react scroll
- Creates components for register flow and components to support the flow
- Enhances Event Card to Suppor a link collection an to support interactive mode
- Fixes a vulnerability 

## How to test
- Create a file at the root of the project with the name .env. This file will be ignored
-Add the following value and save
`REACT_APP_ANALYTICS_ID= EXAMPLEUA121212`
- Run `yarn install`
- Run `yarn start`
- Navigate to /events/post on the browser
- You can create an event card by following the flow
- Check the button participate
